main.css

```diff
@@ -94,15 +94,56 @@ footer
     background-color: #061428;
 }
 
-#sidebar
+#mobileMenuIcon
```

Comment:
Again, we should call this something different since we want it on desktop, too. Maybe `expandIcon` and/or `contractIcon`, `expandContractIcon`, etc.